server/tserver/src/main/java/org/apache/accumulo/tserver/scan/NextBatchTask.java

```diff
@@ -93,10 +106,12 @@ public void run() {
         addResult(iie);
       }
     } catch (TooManyFilesException | SampleNotPresentException e) {
+      recordException(scanSession);
```

Comment:
NextBatchTask run executes normal scans on the server side.  There is another class named LookupTask that executes batch scans on there server side.  Will need to add similar code to it.